apps/storefront/src/pages/order/table/B3Table.tsx

```diff
@@ -0,0 +1,262 @@
+import { ChangeEvent, MouseEvent, ReactElement, ReactNode, useContext } from 'react';
+import { useB3Lang } from '@b3/lang';
+import {
+  Card,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+} from '@mui/material';
+import TableSortLabel from '@mui/material/TableSortLabel';
+
+import { b3HexToRgb, getContrastColor } from '@/components/outSideComponents/utils/b3CustomStyles';
+import { useMobile } from '@/hooks';
+import { CustomStyleContext } from '@/shared/customStyleButton';
+
+import B3NoData from './B3NoData';
+
+interface NodeWrapper<T extends object> {
+  node: T;
+}
+
+export type PossibleNodeWrapper<T extends object> = T | NodeWrapper<T>;
+
+export const isNodeWrapper = <T extends object>(
+  item: PossibleNodeWrapper<T>,
+): item is NodeWrapper<T> => 'node' in item;
```

Comment:
is the plan to remove this in another PR?

Or I guess it's harmless noise, that's not getting in the way of working out contracts 🤷

Reply:
I wasn't gonna remove it for now. I suppose it will vanish when we move from the use of two apis.